saleor/core/tests/test_core.py

```diff
@@ -149,6 +150,30 @@ def test_create_fake_order(db, monkeypatch, image, media_root, warehouse):
     assert Order.objects.all().count() == 2
 
 
+@pytest.mark.parametrize("how_many", [1, 3, 10])
```

Comment:
Do we need to test with such big numbers like 3,5,10? 
It doesn't create its bulk, which means that we will add two parametrized(6 cases in summary) tests which will create over  28 orders. I would recommend checking what kind of object is required to create order from `random.create_orders`.
`test_create_fake_order` is one of the slowest tests in saleor and it creates only two orders.

Reply:
The main problem here is not the orders creation but the `random_data.create_products_by_schema("/", False)` since this is what really takes time here. Will try to do something with it.